src/raptor/RaptorAlgorithm.ts

```diff
@@ -23,7 +23,8 @@ export class RaptorAlgorithm {
     origin: Stop,
     date: number,
     dow: DayOfWeek,
-    time: Time
+    time: Time,
+    notVias: Stop[]
```

Comment:
I would have made the default value here, rather than further up in the API personally.

Reply:
So would you rather have the default here and it required the higher ups or it have a default value all the way up?

Reply:
I would have it default here, then it essentially becomes optional all the way up. It means that the other APIs wouldn't have changed in this instance.
But I'm not too hung up on changing this, given that you have changed all the API sigs anyway.

Reply:
8c8b9d7 Ive added it to be optional here and removed a case of an empty []. The others have renamed as optional defaults where they are needed i beleive

Reply:
Looks good.